### DISCUSSION TITLE
How do I view my lists on the mobile app?

### DISCUSSION THREAD

**Discussion by GroovinChip** [community/community#12177] [Other Feature Feedback, Questions, & Ideas] [answered]
I’ve added several repos to a list, but I can’t actually find and view the list.

**Comment by jsoref** [comment]
Fwiw, it doesn't look like you can view stars (the underlying feature) either.

**Comment by GroovinChip** [comment, marked as answer]
I found it! It’s under “starred” for my profile

**Reply by tang88888888** [reply]
> 我找到了！我的个人资料在“星形”下
